### DISCUSSION TITLE
好奇一個問題  有空的人可以點進來看看

### DISCUSSION THREAD

**Discussion by 123hi123** [moesnow/March7thAssistant#307] [General]
為甚麼在遠端桌面上設定的星鐵參數 例如螢幕分辨律是否開啟聲音 當我回到我本機上的時候設定不會用我在遠端電腦的哪一套 而是用我原本設定的 真的很方便 好奇是作者實現的 還是星鐵貼心的設計
不一定是作者 有想法的人都可以來提個一嘴 滿足一下我的好奇心

**Comment by KiMelody** [comment]
是因为远程桌面上的是另一个计算机用户，自己电脑上会有一个设置的存储，那边会有另一个，这是Windows的特性

**Reply by 123hi123** [reply]
我有做過一個測試
我在現有的兩個用戶的基礎就上，我又在新創一個帳號
所以這個第三個使用者，我再開啟星鐵
發現確實是不會改變
但絕區零就不是了
那如果照你說的Windows特性，為什麼絕區零就在多個用戶上開啟的時候會顯示同樣一套的顯示邏輯

**Reply by KiMelody** [reply]
正常·来说，软件的各种配置是通过配置文件保存在电脑的系统路径下的C:\Users\你的用户名\AppData\路径下，不同账户正常情况下应该拥有着各自的配置文件，但根据你的描述，绝区零不发生改变，可能说明绝区零的配置文件并没有放置在我说的上面的路径里，而是放置在了软件目录里，或者放置在了磁盘上的公用区域，所以 才会不发生改变

**Reply by 123hi123** [reply]
好 我回去就用電腦堅持看看 多謝回覆
